src/ishikawa_backend.erl

```diff
@@ -47,7 +47,11 @@
 
 -record(state, {actor :: actor(),
                 vv :: timestamp(),
-                members :: [term()]}).
+                members :: [term()],
```

Comment:
What is node()? Where is it defined? in vclock.erl, there is vclock_node() that is a term ().